ci-operator/config/openshift/machine-api-provider-ibmcloud/openshift-machine-api-provider-ibmcloud-main.yaml

```diff
@@ -0,0 +1,76 @@
+base_images:
+  base:
+    name: "4.12"
+    namespace: ocp
+    tag: base
+  ocp_builder_rhel-8-golang-1.18-openshift-4.11:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.18-openshift-4.11
+binary_build_commands: NO_DOCKER=1 make build
+build_root:
+  from_repository: true
+images:
+- dockerfile_path: Dockerfile
+  from: base
+  inputs:
+    ocp_builder_rhel-8-golang-1.18-openshift-4.11:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.18-openshift-4.11
+  to: ibmcloud-machine-controllers
+promotion:
+  name: "4.12"
+  namespace: ocp
+releases:
+  initial:
+    integration:
+      name: "4.12"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.12"
+      namespace: ocp
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: unit
+  commands: NO_DOCKER=1 make test
+  container:
+    from: src
+- as: goimports
+  commands: IS_CONTAINER=TRUE NO_DOCKER=1 make goimports
+  container:
+    from: src
+- as: govet
+  commands: IS_CONTAINER=TRUE NO_DOCKER=1 make vet
+  container:
+    from: src
+- as: generate
+  commands: IS_CONTAINER=TRUE NO_DOCKER=1 make generate
+  container:
+    from: src
+- as: golint
+  commands: IS_CONTAINER=TRUE NO_DOCKER=1 make lint
+  container:
+    from: src
+- as: e2e-aws
+  steps:
+    cluster_profile: aws
+    workflow: openshift-e2e-aws
+- as: e2e-aws-serial
+  steps:
+    cluster_profile: aws
+    workflow: openshift-e2e-aws-serial
+- as: e2e-ibmcloud
+  optional: true
```

Comment:
Can the IBM E2E not be required? Eventually we don't want to run AWS here but just IBM

Reply:
I copied it over from [cluster-api-ibmcloud](https://github.com/Karthik-K-N/release/blob/ce875f0a76cc302ef70918013d700cda2dabaae3/ci-operator/config/openshift/cluster-api-provider-ibmcloud/openshift-cluster-api-provider-ibmcloud-main.yaml#L69) directory where its mentioned as optional, may be i can make it as required and remove aws